lib/send-email-verification.js

```diff
@@ -0,0 +1,33 @@
+const config = require('config');
+const sendMail = require('./mail-handler');
+const log = require('./logger');
+
+const HOST = config.get('host');
+const ROUTE_PREFIX = config.get('routePrefix');
+
+const sendEmailVerification = async (user, reply) => {
+  // Send email verification
+  let verificationToken;
+  try {
+    verificationToken = await reply.jwtSign({
+      _id: user._id,
+    }, {
+      expiresIn: '2d',
+    });
+  } catch (error) {
+    log.error('Error trying to create verification token! ', error);
```

Comment:
1. Funktion ajaminen pitäis varmaan lopettaa tässä, miten lähetät mailin ilman sitä tokenia?
2. Melkeimpä ottaisin try catchit pois ja antaisin tämän errorin mennä tätä funktiota kutsuneelle. Try Catchit veks siis

Reply:
Toteuttamalla 2 saat automaattisesti myös 1 =)

Reply:
Jees, eli ei haittaa vaik kaks eri erroria menee samaan try-catchiin sit